### PR TITLE
Feature/accessibility

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
# Description

Added Accessible features: tabbing, keydown selection, and screen reader navigation.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

Ran lighthouse on the site to test.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -281,18 +285,20 @@ function showAvailableRooms(date, type, hotel) {
...
     roomView.addEventListener("click", (event) => showSelectedRoom(event, hotel))
+    roomView.addEventListener("keydown", (event) => showSelectedRoom(event, hotel))
     domUpdates.stringDisplay(bookingHeader, "Rooms available for this date:")
     domUpdates.populateRoomArray(hotel.filterRoomsByAvailability(parsedDate, customer), roomView)
   }
 }
 
 function showSelectedRoom(event, hotel) {
-  if (event.target.closest("article")) {
+  if ((event.target.closest('article') && event instanceof MouseEvent) || event.keyCode === 13) {
```
Comment (tylrs) [inline src/scripts.js:301]: Change to double quotes

File: src/scripts.js [hunk 2]
```diff
@@ -69,13 +62,24 @@ chosenType.value, hotel));
...
+chosenType.addEventListener("keydown", ariaStateChange)
 
 selectedRoom.addEventListener("click", (event) => bookRoom(event, hotel, customer, bookingsData))
 
 ///Fetch stuff here
 window.addEventListener('load', fetchData);
 
+//Aria
+function ariaStateChange() {
+  let attribute = chosenType.getAttribute("aria-expanded");
```
Comment (tylrs) [inline src/scripts.js:75]: I didn't know that get attribute was a method! Nice

**Review by tylrs — APPROVED** [review]

File: src/scripts.js (hunk 1 above)
Comment (AshleyOh-bit) [reply src/scripts.js:301]: Noted. Thank you. @hannahhch

File: src/scripts.js (hunk 2 above)
Comment (AshleyOh-bit) [reply src/scripts.js:75]: Very helpful for screen readers. @hannahhch

**Pull request merged**